.github/workflows/scripts/macos/build-dependencies.sh

```diff
@@ -153,30 +164,26 @@ patch -u src/tools/macdeployqt/shared/shared.cpp <<EOF
  
      // Platforminputcontext plugins if QtGui is in use
 EOF
-cmake -B build -DCMAKE_PREFIX_PATH="$INSTALLDIR" -DCMAKE_INSTALL_PREFIX="$INSTALLDIR" -DCMAKE_BUILD_TYPE=Release -DCMAKE_OSX_ARCHITECTURES="x86_64" -DFEATURE_dbus=OFF -DFEATURE_framework=OFF -DFEATURE_icu=OFF -DFEATURE_opengl=OFF -DFEATURE_printsupport=OFF -DFEATURE_sql=OFF -DFEATURE_gssapi=OFF
+cmake -B build "${CMAKE_COMMON[@]}" -DFEATURE_dbus=OFF -DFEATURE_framework=OFF -DFEATURE_icu=OFF -DFEATURE_opengl=OFF -DFEATURE_printsupport=OFF -DFEATURE_sql=OFF -DFEATURE_gssapi=OFF
 make -C build "-j$NPROCS"
 make -C build install
 cd ..
 
 echo "Installing Qt SVG..."
 tar xf "qtsvg-everywhere-src-$QT.tar.xz"
 cd "qtsvg-everywhere-src-$QT"
-mkdir build
-cd build
-"$INSTALLDIR/bin/qt-configure-module" ..
-make "-j$NPROCS"
-make install
-cd ../..
+cmake -B build "${CMAKE_COMMON[@]}"
+make -C build "-j$NPROCS"
+make -C build install
+cd ..
 
 echo "Installing Qt Image Formats..."
 tar xf "qtimageformats-everywhere-src-$QT.tar.xz"
 cd "qtimageformats-everywhere-src-$QT"
-mkdir build
-cd build
-"$INSTALLDIR/bin/qt-configure-module" ..
-make "-j$NPROCS"
-make install
-cd ../..
+cmake -B build "${CMAKE_COMMON[@]}"
```

Comment:
Is there a specific reason you removed the `qt-configure-module` script? That's the intended way of building qt submodules, if there's additional cmake args that need to be specified, they can be tacked onto the end.

Reply:
Does it mind if I re-specify `CMAKE_INSTALL_PREFIX`, etc?

Reply:
I don't see why not.